controllers/machinehealthcheck_controller.go

```diff
@@ -85,20 +85,20 @@ func (r *MachineHealthCheckReconciler) SetupWithManager(ctx context.Context, mgr
 		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(ctrl.LoggerFrom(ctx), r.WatchFilterValue)).
 		Build(r)
 	if err != nil {
-		return errors.Wrap(err, "failed setting up with a controller manager")
+		return errors.Wrap(err, "failed setting up with a managedController manager")
```

Comment:
The errors that have been changed here should stay as they were IMO, `failed setting up with a controller manager` makes more sense in this context